homeassistant/util/package.py

```diff
@@ -8,6 +8,7 @@
 import logging
 import os
 from pathlib import Path
+import site
```

Comment:
This module shouldn't change. Looks like a rebase gone wrong.

Reply:
Thanks, I hadn't noticed this. Corrected.